### PR TITLE
Create locale.xml

### PR DESCRIPTION
**Pull request by jorgeandr3s**
Spanish translation of plugins/generic/browse/locale/

### CODE REVIEW

**Comment by jorgeandr3s** [comment]
Updated to follow the translation guide. Thanks for the info.

File: plugins/generic/browse/locale/es_ES/locale.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/browse/locale/es_ES/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
+  * Localization strings.
```
Comment (marcbria) [inline plugins/generic/browse/locale/es_ES/locale.xml]: Please, label the translation as contrib as follows:
"Localization strings (contrib)"

**Comment by asmecher** [comment]
Closing these for now, as they're getting old -- @jorgeandr3s, feel free to re-open this if you're still active.

**Comment by marcbria** [comment]
Hi Alec

**Comment by asmecher** [comment]
Thanks, @marcbria, and sorry for the assumption, @jorgeandr3s -- I thought this was abandoned. Merging now. Thanks for contributing!

**Comment by marcbria** [comment]
No need to sorry. Indeed, the ball was in my court. ;-)
@jorgeandr3s , please take a look to your other issues if you want them to be included in the next release.

**Pull request merged**